RecoLocalFastTime/FTLCommonAlgos/plugins/BTLUncalibRecHitAlgo.cc

```diff
@@ -68,11 +74,20 @@ FTLUncalibratedRecHit BTLUncalibRecHitAlgo::makeRecHit(const BTLDataFrame& dataF
     amplitude.second = sampleRight.data() * adcLSB_;
     time.second = sampleRight.toa() * toaLSBToNS_;
 
+    nHits += 1.;
+
     // Correct the time of the right SiPM for the time-walk
     time.second -= timeCorr_p0_ * pow(amplitude.second, timeCorr_p1_) + timeCorr_p2_;
     flag |= (0x1 << 1);
   }
 
+  // --- Calculate the error on the hit time using the provided parameterization
+
+  std::vector<double> amplitudeV = {(amplitude.first + amplitude.second) / nHits};
+  std::vector<double> emptyV;
```

Comment:
did you try using `std::array` here? looks like FormulaEvaluator supports it, and it could be more efficient. Also, I think both could be const?

Reply:
You're right. Done.